### PR TITLE
Animal baseHealthScale and combatPower patch removal

### PR DESCRIPTION
**Pull request by SamaelGray**
## Changes

- Removed the `baseHealthScale` patches for base game and modded animals (except Insectoids).
- Cleaned up some leftover `baseBodySize` patches for animals.
- Removed the combatPower patches of animals to compensate for the removed health scale patches and reduce on their numbers in events.

## Reasoning

- https://discord.com/channels/278818534069501953/700236703180259358/1039484163352436837

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] Playtested a colony
https://discord.com/channels/278818534069501953/322827713335394304/1040325557663051796

### CODE REVIEW

**Review by albertbokor — DISMISSED** [review]

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Will give this a glance.

File: Patches/Core/ThingDefs_Races/Races_Animal_Giant.xml [hunk 1]
```diff
@@ -192,13 +178,6 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="Thrumbo"]/race/baseHealthScale</xpath>
-		<value>
-			<baseHealthScale>5</baseHealthScale>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Races/Races_Animal_Giant.xml:198]: The vanilla Thrumbo basehealthScale is only 0.7. Given the threat they're intended to present, I think the health and combat power patches for it should stay.

File: Patches/Core/ThingDefs_Races/Races_Animal_Temperate.xml [hunk 2]
```diff
@@ -620,11 +550,4 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/PawnKindDef[defName="Tortoise"]/combatPower</xpath>
-		<value>
-			<combatPower>20</combatPower>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Races/Races_Animal_Temperate.xml:626]: Vanilla combat power is 70, meaning one tortoise is equivalent to a little less than a lynx (80), which I think is kind of absurd. I'd probably leave this patch as it is.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

**Comment by N7Huntsman** [comment]
Generally looks alright for the Core animals, with a few bits of specific feedback. Haven't given the mod patches more than a glance, but the seem much the same.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**